### PR TITLE
Adjust build setup

### PR DESCRIPTION
**Pull request by rubenthoms**
* Adjusted build setup (switched to `rollup`, setup bundling of library).
* Installed `Storybook`.
* Switched from demo app to `Storybook` stories (removed unused files).
* Adjusted width and height settings for timeline.

### CODE REVIEW

**Review by howardie — APPROVED** [review]

File: .github/workflows/flow-diagram-explorer.yml [hunk 1]
```diff
@@ -34,9 +34,9 @@ jobs:
               run: |
                   npm run typecheck && npm run lint
 
-            - name: 🤖 Run tests
-              run: |
-                  npm run test
+            #- name: 🤖 Run tests
```
Comment (jsolaas) [inline .github/workflows/flow-diagram-explorer.yml:37]: Should this be added again?

**Review by jsolaas — APPROVED** [review]
lgtm!

File: .github/workflows/flow-diagram-explorer.yml (hunk 1 above)
Comment (rubenthoms) [reply .github/workflows/flow-diagram-explorer.yml:37]: Yes, as soon as we have tests. Since we don't have any yet, `jest` would always fail.

**Review by jorgenherje — APPROVED** [review]
Looks good! 👍

Packed and tested in app, last fix seems to work well!

**Pull request merged**